### ISSUE TITLE
Not Appearing in Sonos or AirPlay

### ISSUE THREAD

**Issue by calucido** [philippe44/AirConnect#45]
I noticed my speakers weren't appearing in AirPlay, so I updated to 0.2.0.2, but now they don't appear in AirPlay, and airupnp does not appear in the "now playing" section of the Sonos app.
Log file:
```
[20:32:58.501313] main:1201 Starting airupnp version: v0.2.0.2 (Jan 30 2018 @ 22:09:20)
[20:32:58.501653] main:1209 no config file, using defaults
[20:32:58.504707] Start:961 Binding to 192.168.1.111:49153
[20:32:58.927528] AddMRDevice:874 [0x697a40]: adding renderer (Back Room)
[20:32:58.936814] MasterHandler:574 [0x697a40]: subscribe success
[20:32:58.959497] ProcessEvent:383 [0x697a40]: UPnP Volume local change 31
[20:32:59.005699] AddMRDevice:874 [0x6995d8]: adding renderer (Bedroom)
[20:32:59.015117] MasterHandler:574 [0x6995d8]: subscribe success
[20:32:59.031508] ProcessEvent:383 [0x6995d8]: UPnP Volume local change 13
```

I rebooted my computer, and updated my speakers (which may actually be the issue), and I'm still getting the same problem and the same log. My computer and speakers are on the same network.

**Comment by philippe44** [comment]
sorry, no idea - the players are properly discovered. I also do not really understand what you mean by "airupnp does not appear in the "now playing" section of the Sonos app"

**Comment by marco79cgn** [comment]
There won't be anything visible in the Sonos App until you start streaming something via AirPlay - which as you said is not possible as the AirPlay targets don't appear. 
Maybe a firewall related issue?

**Comment by calucido** [comment]
It was my fault; airupnp was listening on the server's IP address, which turned out to mean listening on the loopback interface. In my config, I set it to listen to 0.0.0.0 and my devices are now showing up in airplay, but I can never connect to them. I believe this is an issue introduced by updating the speakers, because airupnp was working fine before I updated _them_. I tried downgrading airupnp, and the problem persists; I think all I can do is wait for Sonos to fix their upnp client.
